### PR TITLE
feat: add internationalization i18n

### PR DESCRIPTION
**Pull request by OliynykPro**
## What does this PR do?
Added i18n library and translations

### CODE REVIEW

File: explorer/src/pages/Schema/RecentAttestations/index.tsx [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    SWRKeys.GET_RECENT_ATTESTATION,
+    () => sdk.attestation.findBy(undefined, undefined, { schemaId }),
+    {
+      shouldRetryOnError: false,
+    },
+  );
+  return (
+    <div className="flex flex-col gap-6 w-full px-5 md:px-10">
+      <div className="text-text-primary text-xl font-semibold">{t("attestation.recent")}</div>
+      {/* todo add loading */}
```
Comment (Solniechniy) [inline explorer/src/pages/Schema/RecentAttestations/index.tsx:22]: we've merged PR with skeletons, add here

File: explorer/src/config/translation/index.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+import en from "@/assets/locales/en/en.json";
+
+const resources = {
+  en: {
+    translation: en,
+  },
+};
+
+type TranslationKeys = `${keyof typeof en}`;
```
Comment (voenkomatiwe) [inline explorer/src/config/translation/index.ts:13]: not used, and I think we won’t need it in the future

File: explorer/src/config/translation/index.ts [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  fallbackLng: "en",
+  interpolation: {
+    escapeValue: false,
+  },
+  returnObjects: true,
+  keySeparator: ".",
+  resources,
+});
+
+export type ITranslationKeys = TranslationKeys | `${TranslationKeys}.${string}`;
```
Comment (voenkomatiwe) [inline explorer/src/config/translation/index.ts:26]: and this

File: explorer/src/routes/index.tsx [hunk 4]
```diff
@@ -1,5 +1,6 @@
 import { Route, createHashRouter, createRoutesFromElements } from "react-router-dom";
 
+import "@/config/translation";
```
Comment (voenkomatiwe) [inline explorer/src/routes/index.tsx:3]: why did you decide to do this here?

**Review by voenkomatiwe — APPROVED** [review]

**Review by ars9 — APPROVED** [review]

**Review by Solniechniy — APPROVED** [review]

**Review by orbmis — APPROVED** [review]

**Review by alainncls — APPROVED** [review]

**Pull request merged**